Intly 6595 reportportal integration

wei-lee opened the pull request:
https://issues.redhat.com/browse/INTLY-6595

To review check this commit: 1c891c43adc18b1c6a1c0bd61a9f42eaa680ce09

b1zzu commented on cmd/reportPortalImport.go:
```diff
@@ -0,0 +1,198 @@
+	return false
+}
+
+func readFileFromZip(zipfile string, fileName string) ([]byte, error) {
```
Can we move this to `utils/` I would also need it for Polarion

wei-lee commented:
This PR is ready for review now. To test:

1. Set the following env vars (or add them to `.delorean.yaml` file in your home directory):
   ```
   export RP_TOKEN=<token to access reportportal. See your profile page on reportportal>
   export DELOREAN_AWS_ACCESS_KEY_ID=<access key id for aws account 045835748416>
   export DELOREAN_AWS_SECRET_ACCESS_KEY=<secret key for the same aws account>
   ```
2. Checkout the PR, build it and run:
   ```
   ./delorean report reportportal-import -b rhmi-test-data-storage-tf56 -p <your project name in reportportal>
   ```
3. When it's finished, you should check reportportal and see there should be at least 2 launches created.
4. Run the command again and this time you should see none of them is processed.

b1zzu commented on cmd/reportPortalImport.go:
```diff
@@ -0,0 +1,229 @@
+	defaultImportWorkers = 10
+
+	metadataFileName = "metadata.json"
+	rpTokenKey       = "rp_token"
```
Minor things, can we use the full name instand of the abbreviation `REPORT_PORTAL_TOKEN`, in my opinion, abbreviations are confusing especially if not wider used

wei-lee replied:
okay

b1zzu commented on pkg/reportportal/launch.go:
```diff
@@ -0,0 +1,72 @@
+
+	body := new(bytes.Buffer)
+	writer := multipart.NewWriter(body)
+	part, err := writer.CreateFormFile(importFileFieldName, fi.Name())
```
Instand of using the zip file name we should use the pipeline name in the metadata to upload the test results so that the launcher name will be the pipeline name and not the zip file name 
```suggestion
	part, err := writer.CreateFormFile(importFileFieldName, launchName)
```

wei-lee commented:
@b1zzu I have addressed your comments. Also I added a new `--no-tagging` flag to skip the step of adding tags to the s3 object, makes it easier for our testing.

b1zzu commented:
/lgtm

wei-lee commented:
/approve
